pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -894,7 +1016,21 @@ func (p *PriorityQueue) movePodsToActiveOrBackoffQueue(logger klog.Logger, podIn
 			activated = true
 		}
 	}
-	p.moveRequestCycle = p.schedulingCycle
+
+	inflightPods := sets.New[types.UID]()
+	for uid := range p.inFlightPods {
+		inflightPods.Insert(uid)
+	}
```

Comment:
why are you making a copy of the map?

Reply:
Sorry, it was a remaining thing from the past implementation. I'll remove this.